connector/server/survivalanalysis/survival_query.go

```diff
@@ -146,7 +146,7 @@ func (q *Query) Execute() error {
 			timer = time.Now()
 			logrus.Infof("I2B2 explore for subgroup %d", i)
 			logrus.Tracef("panels %+v", panels)
-			initialCount, patientList, err := SubGroupExplore(q.QueryName, i, panels)
+			_, patientList, err := SubGroupExplore(q.QueryName, i, panels)
```

Comment:
does this fix the patientlist intersection problem? How?

Reply:
because patientList is intersected with the patient list of the cohort, then the initial count is computed as the length of the intersection result.

Reply:
but then `SubGroupExplore` is returning a useless value (the first one)? Would not it be better to just drop it?